net/sshuttle/Makefile

```diff
@@ -0,0 +1,56 @@
+#
+# Copyright (C) 2022 Thomas Cujé
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=sshuttle
+PKG_VERSION:=1.1.1
+PKG_RELEASE:=1
+
+PYPI_NAME:=$(PKG_NAME)
+PKG_HASH:=f5a3ed1e5ab1213c7a6df860af41f1a903ab2cafbfef71f371acdcff21e69ee6
+
+PKG_MAINTAINER:=Thomas Cujé <dev.cuje@freenet.de>
+PKG_LICENSE:=LGPL-2.1-only
+
+include ../../lang/python/pypi.mk
+include $(INCLUDE_DIR)/package.mk
+include ../../lang/python/python3-package.mk
```

Comment:
You should put your package to python folder.

Reply:
I think the application is only losely related to python. Would you place all Java applications in `lang/java/`, too?  `fail2ban` is also using python and not in the python folder. On the functional level sshuttle perfectly relates to network, ssh and I would not look for an application based on the implementation language. Whats the structure behind here?